update_automation_example.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import json
+import demisto
+import unittest
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(
+        description='an example of update automation by demisto.client',
+        usage="""
+  update automation example:
+    %(prog)s -k $DEMISTO_API_KEY -u https://demisto.example.com/ -f my_automation_script.py [-n automation_name]
+  unit test of demisto.client.(Search|Load|Update|Save|Delete)Automation:
+    %(prog)s -t"""
+    )
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-u', '--url', help='Base URL of API', required=True)
```

Comment:
> Note: Required options are generally considered bad form because users expect options to be optional, and thus they should be avoided when possible.

* https://docs.python.org/3/library/argparse.html#required

Maybe these two should be positional arguments instead?

```python
parser.add_argument('key', help='The API key to access the server')
parser.add_argument('url', help='Base URL of API')

```

Reply:
used positional arguments instead of required=True.(e2beb08)